beets/ui/__init__.py

```diff
@@ -399,7 +399,7 @@ def input_select_objects(prompt, objs, rep):
     """
     choice = input_options(
         (u'y', u'n', u's'), False,
-        u'%s? (Yes/no/select)' % prompt)
+        u'%s? (Yes/no/select)' % (prompt_all or prompt))
```

Comment:
I'm a little confused about `prompt` vs. `prompt_all`. Two questions, I guess:

- Is it equivalent to just do `prompt_all or prompt` when *calling* this function, rather than inside the function? That seems a little simpler.
- Since the latter always overrides the former if provided, and both seem to be provided in `remove_items` below, what is the purpose of plain old `prompt`?

Reply:
It's not equivalent to `prompt_all or prompt` when calling, since `prompt` is used again later in `input_select_objects`: `prompt_all`, if given, is the message shown with the initial "Yes/no/select" query, whereas prompt is the message shown for each individual item if "select" is chosen. Maybe `prompt_all` is not a very descriptive name, but I couldn't quickly come up with anything better.

The modify CLI command also uses `input_select_objects`, but doesn't give `prompt_all`. The new argument allowed me to retain the initial prompt that the remove command used to print ("Really DELETE N files"), which however isn't suitable to be repeated for each item upon interactive selection.

Reply:
Got it; I missed that other use of `prompt`! Thanks for clarifying. This totally makes sense.